### DISCUSSION TITLE
runserver_plus doesnt reload on file changes

### DISCUSSION THREAD

**Discussion by tasiotas** [cookiecutter/cookiecutter-django#3640] [Q&A] [answered]
Hi,

Working in Docker, followed guide in docs. All running nicely, except reloading server when I update .py files.

It does work fine if I change "start" command to use classic runserver, but then Im loosing amazing Werkzeug.

This is what I get in log when spinning up container:

```
PostgreSQL is available
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, django_celery_beat, sessions, sites
Running migrations:
  No migrations to apply.
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://172.19.0.4:8000/ (Press CTRL+C to quit)
 * Restarting with watchdog (inotify)
Attached!
Performing system checks...

System check identified no issues (0 silenced).

Django version 3.2.12, using settings 'config.settings.local'
Development server is running at http://0.0.0.0:8000/
Using the Werkzeug debugger (http://werkzeug.pocoo.org/)
Quit the server with CONTROL-C.
 * Debugger is active!
 * Debugger PIN: 128-778-450
```
It looks like watchdog is there. Is there any configuration or logs I should check?

Thank you

**Comment by foarsitter** [comment, marked as answer]
On which platform you are using Docker?

**Reply by tasiotas** [reply]
Windows

**Reply by foarsitter** [reply]
Windows and Docker in WSL2 mode I guess? I think your solution can be found here: https://levelup.gitconnected.com/docker-desktop-on-wsl2-the-problem-with-mixing-file-systems-a8b5dcd79b22

**Reply by tasiotas** [reply]
thank you, excellent article. Over the weekend I realized its because of cross platform file systems. I will switch to pooling method in Werkzeug, as somehow not feeling like jumping into WSL dev workflow on Windows.

Thanks
